src/box/system_allocator.h

```diff
@@ -0,0 +1,181 @@
+#ifndef TARANTOOL_BOX_SYSTEM_ALLOCATOR_H_INCLUDED
+#define TARANTOOL_BOX_SYSTEM_ALLOCATOR_H_INCLUDED
+/*
+ * Copyright 2010-2016, Tarantool AUTHORS, please see AUTHORS file.
```

Comment:
I know 2020 is bad, but we can't simply return to 2016 like that.

Reply:
Fixed